test/db/formats/elf/elf-relro

```diff
@@ -131,7 +131,7 @@ nth vaddr      bind   type   lib name
 104 0x00052280 GLOBAL FUNC       sigemptyset
 105 0x00052288 GLOBAL FUNC       ftell
 106 0x00052290 GLOBAL FUNC       feof
-107 0x00443a32 WEAK   NOTYPE     __gmon_start__
+107 0x00000000 WEAK   NOTYPE     __gmon_start__
```

Comment:
imho this should be -1

Reply:
The problem is that in all tests when get_import_addr can't find any addr the value is set to 0.
It is strange to change this behavior.